mcm/reader.py

```diff
@@ -233,7 +232,7 @@ def num_columns(self):
 
     def headers(self):
         """original ordered list of spreadsheet headers"""
-        return self.csvreader.fieldnames
+        return self.csvreader.unicode_fieldnames
```

Comment:
In this case the headers are read fine, this is where the badness gets introduced, i.e. instead of transliterating (ft²) to (ft2) we get (ft�)